### DISCUSSION TITLE
Bottom Enhanced Mixing

### DISCUSSION THREAD

**Discussion by msdevana** [CliMA/Oceananigans.jl#2755] [General]
Hi Oceananigans Community!

I am just starting out playing with Oceananigans and trying to do some experiments examining bottom enhanced mixing . 
Any tips on how to specify a bottom enhanced diffusivity profile rather than a constant value?

Thanks!
Manish

**Comment by glwagner** [comment]
From the [docstring for ScalarDiffusivity](https://github.com/CliMA/Oceananigans.jl/blob/1a288c175d07b3d4262e965c7e75376d5541e24d/src/TurbulenceClosures/turbulence_closure_implementations/scalar_diffusivity.jl#L13-L42):

```
`ν` and the fields of `κ` may be constants (converted to `FT`), arrays, fields or
  - functions of `(x, y, z, t)` if `discrete_form = false`
  - functions of `(i, j, k, grid, LX, LY, LZ)` with `LX`, `LY` and `LZ` are either `Face()` or `Center()` if
    `discrete_form = true`.
```

So, you can use a function with a bottom-enhanced shape like 

```julia
@inline κ(x, y, z, t) = κ0 + κ1 * exp(-(z + H)^2 / (2*d^2))
```

where `z=-H` is the bottom and `d` is the decay scale of the region of enhanced mixing. Or, you can provide an array with discrete values for the diffusivity.

**Reply by msdevana** [reply]
Thank you! Super helpful and straightforward
